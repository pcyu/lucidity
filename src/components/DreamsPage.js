import React from 'react'
import DreamList from './DreamList'
import DreamListFilters from './DreamListFilters'

const DreamsPage = () => (
  <div>
    <DreamListFilters />
    <DreamList />
  </div>
)

export default DreamsPage