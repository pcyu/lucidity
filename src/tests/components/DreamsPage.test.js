import React from 'react'
import { shallow } from 'enzyme'
import DreamsPage from '../../components/DreamsPage'

test('should render DreamsPage correctly', () => {
  const wrapper = shallow(<DreamsPage />)
  expect(wrapper).toMatchSnapshot()
})