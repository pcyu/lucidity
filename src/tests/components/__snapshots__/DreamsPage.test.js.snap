// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render DreamsPage correctly 1`] = `
<div>
  <Connect(DreamListFilters) />
  <Connect(DreamList) />
</div>
`;
